app.js

```diff
@@ -0,0 +1,76 @@
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+const bookFormInput = document.querySelector('#book_form');
+const button = document.querySelector('#btn');
+
+button.addEventListener('click', (event)=> {
+event.preventDefault();
+    if(titleInput.value == "" && authorInput) {
+        return
+    }
+
+    else {
+        const booklist = document.querySelector('#book-list')
+        const newbook = document.createElement('div');
+        const newAuthor = document.createElement('div');
```

Comment:
- [ ] Let's  us a function to add a new book to the collection, with title and author. 
`Hint`: For book collection you can use an array of objects.
- [ ] Also `localStorage` to preserve data in the browser's memory.